packages/makeup-active-descendant/src/index.js

```diff
@@ -121,37 +122,31 @@ class LinearActiveDescendant extends ActiveDescendant {
         });
 
         if (this._options.autoInit > -1) {
-            const itemEl = this.filteredItems[this._options.autoInit];
+            const itemEl = this.items[this._options.autoInit];
 
             itemEl.classList.add(this._options.activeDescendantClassName);
 
             this._focusEl.setAttribute('aria-activedescendant', itemEl.id);
         }
     }
 
+    // todo: rename or remove
     get index() {
         return this._navigationEmitter.model.index;
     }
 
+    // todo: rename or remove
     set index(newIndex) {
         this._navigationEmitter.model.index = newIndex;
     }
 
+    // todo: rename
     reset() {
         this._navigationEmitter.model.reset();
     }
 
-    get filteredItems() {
-        return this._navigationEmitter.model.filteredItems;
-    }
-
     get items() {
-        return this._navigationEmitter.model.items;
-    }
-
-    // backwards compat
-    get _items() {
-        return this.items;
+        return this._el.querySelectorAll(`${this._itemSelector}:not([hidden])`);
```

Comment:
I understand that this change allows for alternative filtering of items and it likely won't hurt in terms of efficiency because `_items` is only really accessed `onClick`, but the repeated query selector feels like a bit of a code smell to me. Maybe it would be more effective to use
```js
this._navigationEmitter.model.items.filter((el) => /* additional filters */);
```